onedal/linear_model/incremental_linear_model.py

```diff
@@ -0,0 +1,156 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import warnings
+
+import numpy as np
+
+from daal4py.sklearn._utils import get_dtype
+
+from ..common.hyperparameters import get_hyperparameters
+from ..datatypes import _convert_to_supported, from_table, to_table
+from ..utils import _check_X_y, _num_features
+from .linear_model import BaseLinearRegression
```

Comment:
```suggestion
from daal4py.sklearn._utils import get_dtype
from onedal.common.hyperparameters import get_hyperparameters
from onedal.datatypes import _convert_to_supported, from_table, to_table
from onedal.utils import _check_X_y, _num_features
from onedal.linear_model import BaseLinearRegression
```
I would prefer we avoid relative imports if possible

Reply:
it contradicts with sklearn developer guide